docs/pages/1712/solution/building.html

```diff
@@ -116,15 +116,39 @@ <h2><a href="#">Building OSTRICH</a></h2>
 							<p align="left">Initial Publication:</p>
 							<time class="time" datetime="2015-11-01">September 16, 2020</time>
 							<p align="left">Modified:</p>
-							<time class="time" datetime="2015-11-01">September 16, 2020</time>
+							<time class="time" datetime="2015-11-01">September 29, 2020</time>
```

Comment:
Why is `datetime`  Nov 1, 2015 and date given Sep 29, 2020?

Reply:
I think that's part of the template since it's in the tag.